integrations/aerospike/ops_agent_metadata.yaml

```diff
@@ -0,0 +1,66 @@
+platforms:
+- type: GCE
+  launch_stage: GA
+  install_documentation_url: https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/aerospike
```

Comment:
This PR is triggered by our doc generator when @mtabasko started working on the public doc. 

This specific line will be where the new integration public doc posted, currently still 404 as it's WIP not being released out yet. 

As for why this PR involves so much changes not just the public url is because https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/232 was merged later than this current PR. When comparing https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/232 with this current PR, the only difference is this PR contains everything the other PR has + the installation url and launch_stage part in addition. This PR is like a superset.